.gitignore

```diff
@@ -1,16 +1,6 @@
-.gradle
```

Comment:
I guess this is related to swapping from gradle to maven? Definitely pedantic but I'd prefer a separate commit for this. Can be the same PR.

Reply:
Yea, that's correct. I'll split it out into a separate commit.

Reply:
Thank you!